### ISSUE TITLE
Issues on GIthub

### ISSUE THREAD

**Issue by nsfphyman** [GT-Nexus/AMOS-Gateway#48]
I am confused about the fact that people are posting issues but no one is being assigned and I see no way to actually respond to these issues.  What am I missing.  This is not the way I am used to doing this on the DLMF site.

**Comment by spamidig** [comment]
Do you see this "Add a comment" frame below the issue..

**Comment by nsfphyman** [comment]
Yes, I see that but there is way to use the right hand col.  I cannot assign, label or do anything.  On the DLMF site those are clickable objects that can be used.

From: Sudhakar Pamidighantam ***@***.***>
Reply-To: GT-ARTISAN/AMOS-Gateway ***@***.***>
Date: Tuesday, February 6, 2024 at 1:16 PM
To: GT-ARTISAN/AMOS-Gateway ***@***.***>
Cc: "Schneider, Barry I. (Fed)" ***@***.***>, Author ***@***.***>
Subject: Re: [GT-ARTISAN/AMOS-Gateway] Issues on GIthub (Issue #48)


Do you see this "Add a comment" frame below the issue..
